src/OpenTelemetry.Exporter.Geneva/GenevaLoggingExtensions.cs

```diff
@@ -24,13 +24,22 @@ namespace Microsoft.Extensions.Logging;
 
 public static class GenevaLoggingExtensions
 {
-    public static OpenTelemetryLoggerOptions AddGenevaLogExporter(this OpenTelemetryLoggerOptions options, Action<GenevaExporterOptions> configure)
+    public static OpenTelemetryLoggerOptions AddGenevaLogExporter(
+        this OpenTelemetryLoggerOptions options,
+        Action<GenevaExporterOptions> configure,
+        Func<BaseExporter<LogRecord>, BaseProcessor<LogRecord>> newProcessorDelegate = null)
```

Comment:
Which processor is used by the Exporter is an internal detail for this Exporter, and we don't want to expose more options for users to play with. The exporter relies on the specific type of processor being in the pipeline, and we don't expect end users to tweak it.

Reply:
@cijothomas - please make it work reliably. Presently event loss is blocking us. This is a strong reason. I believe that Processors are intentionally exposed to devs working on custom exporters. If the fix to expose a delegate isn't the best fix, then could we please at least have the settings to configure the **batching** and the **flush** timers?

Reply:
See https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/925 for ability to configure batch processor options. Only added for Traces now, will come for Logs in the 1.5.0 version.

Reply:
When can we expect something like this added to Geneva Logs exporter?

Reply:
after 1.4 release only.